avogadro/rendering/camera.h

```diff
@@ -120,7 +120,21 @@ class AVOGADRORENDERING_EXPORT Camera
    * @param zFar is the distance from the viewer to the far clipping plane.
    */
   void calculatePerspective(float fieldOfView, float zNear, float zFar);
-
+#ifdef _3DCONNEXION
```

Comment:
This seems like a perfectly reasonable API addition. In that case, please do not `#ifdef` the code and put the implementation properly in `camera.cpp`.